qiskit/__init__.py

```diff
@@ -96,9 +96,10 @@ def __bool__(self):
 
                 self.aer = aer.Aer
                 warnings.warn(
-                    "The qiskit.Aer entry point is deprecated instead you should use this "
+                    "The qiskit.Aer entry point will be deprecated in a future release and "
+                    "will subsequently removed after that. Instead you should use this "
```

Comment:
```suggestion
                    "subsequently removed. Instead you should use this "
```
(or "will subsequently be removed" with the extra "be", but either way, "subsequently" and "after that" is redundant)